frame/composable-traits/src/assets.rs

```diff
@@ -23,7 +23,7 @@ impl BasicAssetMetadata {
 	}
 }
 
-#[derive(Decode, Encode, Debug, Clone, PartialEq)]
+#[derive(Decode, Encode, Debug, Clone, PartialEq, Eq)]
```

Comment:
Curious to know if these were auto-fixes by `clippy` or manual additions by you

Reply:
Auto-fixes when running `clippy --fix`.

I was curious about why adding this derive too, here is the [explanation that clippy gives](https://rust-lang.github.io/rust-clippy/master/index.html#derive_partial_eq_without_eq)